modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/Layout/LayoutScreen/ObjectLayoutTabs.tsx

```diff
@@ -31,6 +31,22 @@ const ObjectLayoutTabs: React.FC<React.HTMLAttributes<HTMLElement>> = () => {
 		onClose: () => setVisibleModal(false),
 	});
 
+	const handleAddLayoutBox = (
+		boxType: 'categorization' | 'seo',
+		tabIndex?: number | undefined
```

Comment:
What is the case where this function will be called using an undefined `tabIndex`?